### DISCUSSION TITLE
Docs indicate that `IntegerField` takes a `choices` parameter

### DISCUSSION THREAD

**Discussion by rmcloughlin** [encode/django-rest-framework#9597] [Potential Issue] [answered]
I'm reading the docs, and in [this section](https://www.django-rest-framework.org/api-guide/serializers/#validators) the second example contains this line:
`room_number = serializers.IntegerField(choices=[101, 102, 103, 201])`
But if I try to run that code I am shown:
`TypeError: Field.__init__() got an unexpected keyword argument 'choices'`

**Comment by browniebroke** [comment, marked as answer]
This is already fixed in the docs source (fixed in https://github.com/encode/django-rest-framework/pull/9563):

https://github.com/encode/django-rest-framework/blob/9016efe3fc412488df92912c619f8f24fed2937c/docs/api-guide/serializers.md?plain=1#L236

So I assume it's a deployment issue, we usually update them when we make a release.
